x/checkers/types/full_game_test.go

```diff
@@ -97,7 +97,7 @@ func TestParseGameWrongTurnColor(t *testing.T) {
 	storedGame.Turn = "w"
 	game, err := storedGame.ParseGame()
 	require.Nil(t, game)
-	require.EqualError(t, err, "game cannot be parsed: Turn: w")
+	require.EqualError(t, err, "game cannot be parsed: turn: w")
```

Comment:
Oh wow. Thanks.
Because this repo has a very clear commit lineage, created by reordering and squashing, I will have to include the fix myself at the right branch, and force push all the subsequent ones.

Reply:
https://github.com/cosmos/b9-checkers-academy-draft/commit/ceb8cb75195e86dc95ba1645cf7be51f1e0e9e57

Reply:
Oh does that mean that each section has its own branch? I could have added the fix on that branch then, if you would have kindly pointed it out, then made a PR for that branch to main? In any case, done now.

Reply:
If you had added the fix to the branch, I would have had to squash your commit with the other _meaty_ commit of this branch, and then `cherry-pick` all the other ones on top before force pushing all of them.
What I did instead, which is marginally faster, is `cherry-pick` your commit, reorder it to right after the existing commit that it is fixing, squash the 2 and force push all branches upwards.

Reply:
That's why the timestamps on commits are all wonky.

Reply:
Plus merge commits cannot be squashed.

Reply:
oh nice, neat trick! thanks for sharing.